src/commands/createWebApp/setAppWizardContextDefault.ts

```diff
@@ -43,7 +44,8 @@ export async function setAppWizardContextDefault(wizardContext: IAppServiceWizar
 
     if (!wizardContext.advancedCreation) {
         if (!wizardContext.location) {
-            await LocationListStep.setLocation(wizardContext, 'centralus');
+            const config: AzConfig = await readAzConfig(wizardContext, AzConfigProperties.location);
+            await LocationListStep.setLocation(wizardContext, config.location ? config.location : 'centralus');
         }
 
         if (!wizardContext.newPlanSku) {
```

Comment:
Are there az cli defaults for sku, runtime, or OS?

Reply:
There are not